Essentials/src/plugin.yml

```diff
@@ -527,3 +527,1183 @@ permissions:
+    default: false
+  essentials.fly.others:
+    description: Allows you to toggle fly for other players.
+    default: false
+  essentials.fly.safelogin:
+    description: Players with this command will automatically switch to fly mode if they login whilst floating in the air.
+    default: false
+  essentials.gamemode:
+    description: Allow access to the /gamemode command.
+    default: false
+  essentials.gamemode.others:
+    description: Allows you to change the gamemode of other players.
+    default: false
+  essentials.itemspawn.meta-chapter-[chapter]:
+    description: Allows spawning of specific books only, from book.txt.
+    default: false
+  essentials.enchantments.[enchantmentname]:
+    description: Allows access to each enchantment type.
+    default: false
+  essentials.enchantments.allowunsafe:
+    description: If enabled in the config, this permission allows unsafe enchantments.
+    default: false
+  essentials.give:
+    description: Allow access to the /give command.
+    default: false
+  essentials.itemspawn.exempt:
+    description: Allows spawning of items on the spawn blacklist.
+    default: false
+  essentials.itemspawn.item <itemname>:
+    description: If permission based item spawn: Spawn <itemname.
+    default: false
+  essentials.itemspawn.item [itemid]:
+    description: If permission based item spawn: Spawn [itemid.
+    default: false
+  essentials.itemspawn.item all:
+    description: If permission based item spawn: Spawn all items
+  essentials.itemspawn.meta author:
+    description: Allow author meta to be used in item spawning.
+    default: false
+  essentials.itemspawn.meta book:
+    description: Allows spawning of books with pre filled content from books.txt
+  essentials.itemspawn.meta firework:
+    description: Allow specific meta for fireworks.
+    default: false
+  essentials.itemspawn.meta head:
+    description: Allows spawning of mob heads.
+    default: false
+  essentials.itemspawn.meta lore:
+    description: Allow lore meta to be used in item spawning.
+    default: false
+  essentials.itemspawn.meta title:
+    description: Allow title meta to be used in item spawning.
+    default: false
+  essentials.oversizedstacks:
+    description: Allows spawning of stacks above normal max stack size
+  essentials.god:
+    description: Allow access to the /god command.
+    default: false
+  essentials.god.others:
+    description: Allows you to god other players
+  essentials.god.pvp:
+    description: Allows you to attack other players while in god mode.
+    default: false
+  essentials.hat:
+    description: Allow access to the /hat command.
+    default: false
+  essentials.heal:
+    description: Allow access to the /heal command.
+    default: false
+  essentials.heal.cooldown.bypass:
+    description: Bypass the heal cooldown
+  essentials.heal.others:
+    description: Allows healing another player
+  essentials.itemspawn.meta chapter [chapter]:
```

Comment:
As far as I can tell, this should have dashes (unless YAML breaks if they are added).